app/Model/Vote/Vote.php

```diff
@@ -16,10 +16,10 @@ class Vote extends Aggregate
 {
     /**
      * @ORM\Id()
-     * @ORM\GeneratedValue(strategy="AUTO")
-     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="UUID")
```

Comment:
IIRC tohle pouzije na MySQL `UUID()`, ktere vraci UUID verze 1, ktere obsahuje zakodovany timestamp - tedy vycist z toho poradi take pujde.

Reply:
Díky, to jsem si nevšiml. Už opraveno na UUIDv4